inji-web/src/__tests__/components/DataShare/CustomTimes/CustomExpiryTimesHeader.test.tsx

```diff
@@ -0,0 +1,22 @@
+import {render, screen} from "@testing-library/react";
+import {CustomExpiryTimesHeader} from "../../../../components/DataShare/CustomExpiryTimes/CustomExpiryTimesHeader";
+
+describe("Test the Layout of the Custom Expiry Header", () => {
+
+    beforeEach( ()=> {
+        render(<CustomExpiryTimesHeader title={"CTHeader"} />);
+    } )
+
+    test("Test the presence of the Outer Container", ()=>{
+        const document = screen.getByTestId("CustomExpiryTimesHeader-Outer-Container");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test the presence of the Title Content", ()=>{
+        const document = screen.getByTestId("CustomExpiryTimesHeader-Title-Content");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test to Have the content", ()=>{
+        const document = screen.getByTestId("CustomExpiryTimesHeader-Title-Content");
+        expect(document).toHaveTextContent("CTHeader");
+    })
```

Comment:
One snapshot test should be good enough to test layout with style

Reply:
Snapshot Test we will take it up as part of different card mentioned.